pages/common/ansible.md

```diff
@@ -0,0 +1,27 @@
+# ansible
+
+> Manage groups of computers remotely over SSH
+
+- Run command:
+
+`ansible {{group}} -m command '{{command}}'`
+
+- Run command with sudo:
```

Comment:
A lot of admin-related commands will require sudo.

This one is a mutation of `Run command`, thus placed right after it for quick lookup.